src/service/tables/governance_history.h

```diff
@@ -13,4 +13,9 @@ namespace ccf
   {
     static constexpr auto GOV_HISTORY = "public:ccf.gov.history";
   }
+  using COSEGovernanceHistory = ServiceMap<MemberId, std::vector<uint8_t>>;
+  namespace Tables
+  {
+    static constexpr auto COSE_GOV_HISTORY = "public:ccf.gov.cose_history";
```

Comment:
I believe this table needs to be documented and we should also have an end-to-end audit test for this (`governance_history.py`).

Reply:
Yes!